tests/backup/backup_portworx_test.go

```diff
@@ -727,14 +731,23 @@ var _ = Describe("{ResizeVolumeOnScheduleBackup}", func() {
 			for backupLocationUID, backupLocationName := range backupLocationMap {
 				Step("Getting size of volume before resizing", func() {
 					log.InfoD("Getting size of volume before resizing")
+					label, err := GetAppLabelFromSpec(AppContextsMapping[namespace])
+					dash.VerifyFatal(err, nil, fmt.Sprintf("unable to get the label from the application spec %s", AppContextsMapping[namespace].App.Key))
+					log.Infof("Pod label from the spec %s", label)
```

Comment:
Don not print the whole `label` map. It can be huge

Reply:
Done, removed it was just added for debug purpose